pkg/datapath/linux/node.go

```diff
@@ -1306,7 +1306,11 @@ func (n *linuxNodeHandler) NodeConfigurationChanged(newConfig datapath.LocalNode
 		}
 	}
 
-	return errs
+	if errs != nil {
+		n.log.Warn("node configuration changed but was unable to update node datapath configuration", errs)
```

Comment:
nit: While errors typically should start with lowercase since they are often concatenated into longer message strings in the final output, log messages should start with a capital letter as they are the primary message that is output to the user without any prefix.

Reply:
Separately and maybe a little more substantive for user experience, just thinking about the user-facing presentation of this error message. Do you think we could draw any conclusions for the user about what the potential impact might be? Mitigations or what the user should do in response to this message?

Example:
```suggestion
		n.log.WithFields(logrus.Fields{}).Warn("Node configuration changed but was unable to update node datapath configuration. Local node misconfiguration may lead to unexpected connectivity disruption", errs)
```

(I guess the target for this log message is to try to resolve a segfault on shutdown? But in theory any of the errors from the called function could result in this path being hit, so the error message should cover those cases too..)